bin/launch-process

```diff
@@ -122,8 +123,9 @@ launch_master() {
   fi
 
   # use a default Xmx value for the master
-  local res="$(contains "${ALLUXIO_MASTER_JAVA_OPTS}" "Xmx")"
-  if [[ "${res}" -eq "0" ]]; then
+  local contain_xmx="$(contains "${ALLUXIO_MASTER_JAVA_OPTS}")"
```

Comment:
should this be `local contain_xmx="$(contains "${ALLUXIO_MASTER_JAVA_OPTS}" "Xmx")"`? i'm doing a refactor and the modified line doesn't make sense to me because the contains check doesn't have a second argument anymore @ssz1997

Reply:
:( very true

Reply:
its not a bug if no one notices 🪲

Reply:
I'm on it